modules/auxiliary/admin/chromecast/chromecast_youtube.rb

```diff
@@ -48,7 +51,7 @@ def run
     when 200
       print_status('Stopping video')
     when 404
-      print_error("Couldn't #{action.name.downcase} video")
+      print_error("#{action.name} failed: DIAL unsupported")
```

Comment:
```suggestion
      print_error("#{action.name} failed: CASTV2 unsupported")
```

Reply:
Both are true. The latter is more appropriate going forward.

Reply:
Ha. Both are true. CASTV2 unsupported by the module. DIAL unsupported API?

Reply:
Yep, DIAL is no longer a supported API. The module still supports it, but that basically means the module doesn't work, since Chromecasts are updated silently via the Google mothership.

Let me reword this error message once more, since I tried to keep it terse, and it's not doing users any favors. The module description explains it better.

Reply:
'Error: Read module description' 😄

Reply:
There is a bit of a gray area here, hence my apprehension. I'm not sure if Google is deprecating the entire protocol in Chromecast or just this part of the API. Who knows what Google does and why. What I do know is that CASTV2 is the only way that works now.

Reply:
Request via DIAL protocol failed. CASTV2 is unsupported by this module.

Reply:
I reworded it. Happy with it now.

Reply:
> Once more, with feeling

Great commit 👍